src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,275 @@
+(where typically the positions and orientations of objects derived from the video data are formatted in BIDS, but not the raw camera footage).
+In this specification, positions (and their time derivatives) are represented as Cartesian coordinates along up to three spatial axes,
+and orientations (and their time derivatives) are represented as Euler angles.
+However, to cover recordings from computer graphics applications (for example, virtual 3D motion or immersive virtual reality recording in physical space),
+orientations are also allowed to be represented as quaternions.
+In this case, the quaternion channels can be distinguished from channels containing Euler angles based on the entries in columns `component` and `units` in the `*_channels.tsv` file.
+See subsection on `Channels description` for further details.
+
+Motion data from one tracking system MUST be stored in a `*_motion.tsv` file.
+A tracking system is defined as a group of motion channels that share hardware properties (the recording device) and software properties (the recording duration and sampling rate).
+For example, if the position time series of multiple optical markers is processed via one recording unit, this can be defined as a single tracking system.
+Note that it is not uncommon to have multiple tracking systems to record at the same time.
+Each tracking system should have its own `*tracksys-<label>_motion.tsv` file.
+One column in the `*tracksys-<label>_motion.tsv` file is intended to represent one data channel.
+The ordering of columns has to match the order of rows in the `*channels.tsv` file for unambiguous assignment.
+All relevant metadata about a tracking systems is stored in accompanying sidecar `*tracksys-<label>_motion.json` file.
+The source data from each tracking system in their original format, if different from `.tsv`,
+can be stored in the [`/sourcedata` directory](../common-principles.md#source-vs-raw-vs-derived-data).
+The original data format MAY hold more metadata than currently specified in the `*_motion.json` file.
+
+When multiple tracking systems are used to record motion or motion capture is used alongside the recording of other BIDS modalities, it is possible to temporally synchronise the recordings.
+A guideline to time synchronization between multiple modalities using recording onset and event time offset is described later in the specifications.
+To store the differences between recording onsets, `scans.tsv` files can be used.
+
+To store events which relate to a tracking system, it is recommended to use designated events files per tracking system.
+Such an events file name would include the tracksys key and look like `sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_events.tsv`.
+The onsets in the event file can be related to the starting time of the tracking system in the `scans.tsv` file.
+
+### Sidecar JSON (`*_motion.json`)
```

Comment:
Small suggestion as I am also trying to implement it in the other datatype in another PR: https://github.com/bids-standard/bids-specification/pull/1397/files

Do you think you could split the metadata in several sub-tables:

- hardware
- task information
- institution information
- motion specific metadata

Reply:
Oh! I'd love to see that! I always thought these fields were a bit remixed, so orgnaizing them like that would totally make sense to me :)

Reply:
@Remi-Gau I changed the organization, but I think this is the reason for the new schemacode CI error. Any idea on how to fix this?

Reply:
Made a separate comment for a tiny error. But looks good otherwise.